example/main.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:cassowary/cassowary.dart';
+
+void main() {
+  var solver = Solver();
+  var left = Param(10);
+  var right = Param(20);
+  var widthAtLeast100 = right - left >= cm(100.0);
+  var edgesPositive = (left >= cm(0.0))..priority = Priority.weak;
+  solver
```

Comment:
I think I would remove the trailing comma in the short list, which gives us better dartfmt output here:

```
  solver
    ..addConstraints([widthAtLeast100, edgesPositive])
    ..flushUpdates();
```

Reply:
Done